charts/templates/feast-database/deployment.yaml

```diff
@@ -0,0 +1,77 @@
+{{ $attendees := (.Values.attendees | int) }}
+{{- range $attendee := until $attendees }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mlops-feature-store
```

Comment:
do you want to call it mlops-feature-store while we are in the inner loop? I mean it's totally alright but I thought would it be confusing as this pod will be there from the start and we won't have much MLOps conversation till 3rd section..just a though 🙈